version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "1.2.0"
+ThisBuild / version := "1.7.3"
```

Comment:
This was out of sync with the value in [project/build.properties](https://github.com/guardian/anghammarad/blob/main/project/build.properties).